x-pack/plugins/task_manager/server/task.ts

```diff
@@ -131,6 +137,10 @@ export const taskDefinitionSchema = schema.object(
      * A brief, human-friendly title for this task.
      */
     title: schema.maybe(schema.string()),
+    /**
+     * Priority of this task type. Defaults to "NORMAL" if not defined
+     */
+    priority: schema.maybe(schema.number()),
```

Comment:
I did not do a default value because the scripted sort will only include the priority value if explicitly set in the task definition (and default to 50 if not set) so I thought we could reduce the size of the script that's generated. But I think adding a custom validator that validates the priority is one of the expected values is a good idea!

Reply:
Was actually feeling the same about the default value, after seeing the rest of the code. 👍🏻

Reply:
Added custom validator in https://github.com/elastic/kibana/pull/175334/commits/50d155ac50e6a480baed6d688cad68be1e86b12b